.github/workflows/Dev builds.yml

```diff
@@ -0,0 +1,117 @@
+name: Dev Builds
```

Comment:
Maybe rename to docker_dev and docker_dev.yml

Reply:
have another pr that will utilis the naming as thats what github displays the name as . will rename the file.

Reply:
can probably change it after my next PR as then you can see why it's named like that. It needs a nice name.